config/application.rb

```diff
@@ -58,6 +58,15 @@ class Application < Rails::Application
     config.eager_load_paths << Rails.root.join('lib')
     config.eager_load_paths << Rails.root.join('app', 'validators')
 
+    # registry config needed
+    config.autoloader = :classic
+    registry_cert = "-----BEGIN PRIVATE KEY-----
```

Comment:
Can we use the RMT server's private key here? The we can have the registry use the rmt-ca.crt for verification.